spark-iotdb-connector/pom.xml

```diff
@@ -97,22 +74,31 @@
         <dependency>
             <groupId>org.apache.spark</groupId>
             <artifactId>spark-core_2.11</artifactId>
+            <version>${spark.version}</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.apache.spark</groupId>
             <artifactId>spark-sql_2.11</artifactId>
+            <version>${spark.version}</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.scala-lang</groupId>
             <artifactId>scala-library</artifactId>
+            <version>${scala.version}</version>
         </dependency>
         <dependency>
             <groupId>org.scalatest</groupId>
             <artifactId>scalatest_2.11</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>net.jpountz.lz4</groupId>
+            <artifactId>lz4</artifactId>
+            <version>1.3.0</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
lz4 in master branch has upgraded to 1.8.0, better to keep the same version.

BTW, what this dependency use for here?

Reply:
It is confilcted with the Spark, so it is marked by `provided`.

Reply:
> 

Yep, it throwed an error `java.lang.NoSuchMethodError: net.jpountz.lz4.LZ4BlockInputStream.<init>`.

Reply:
> 

I merged the code, and delete the dependency of the spark pom. It works now.